lib/elixir/lib/calendar/naive_datetime.ex

```diff
@@ -626,8 +626,6 @@ defmodule NaiveDateTime do
         value
       {:error, :incompatible_calendars} ->
         raise ArgumentError, "cannot convert #{inspect naive_datetime} to target calendar #{inspect calendar}, reason: #{inspect naive_datetime.calendar} and #{inspect calendar} have different day rollover moments, making this conversion ambiguous"
-      {:error, reason} ->
```

Comment:
this clause was unused, currently the only possible error is `{:error, :incompatible_calendars}`